### PR TITLE
Saas 12.5 mergesprint jem

### PR DESCRIPTION
**Pull request by jem-odoo**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by jem-odoo** [comment]
@robodoo rebase r+

**Comment by Feyensv** [comment]
@robodoo rebase-ff

File: addons/project_timesheet_holidays/models/hr_holidays.py [hunk 1]
```diff
@@ -65,29 +65,34 @@ def _validate_leave_request(self):
...
         return super(Holidays, self)._validate_leave_request()
 
+    def _timesheet_create_lines(self):
+        self.ensure_one()
+        work_hours_data = self.employee_id.list_work_time_per_day(
+            self.date_from,
+            self.date_to,
+        )
+        for index, (day_date, work_hours_count) in enumerate(work_hours_data):
+            self.env['account.analytic.line'].create(self._timesheet_prepare_line_values(index, work_hours_data, day_date, work_hours_count))
```
Comment (Feyensv) [inline addons/project_timesheet_holidays/models/hr_holidays.py]: @jem-odoo a sudo() is missing here ;)

**Comment by jem-odoo** [comment]
@robodoo rebase-ff r+

**Pull request closed without merging**